src/Finch_SolidificationData.hpp

```diff
@@ -265,6 +266,14 @@ class SolidificationData
         }
 
         fout.close();
+
+        MPI_Barrier( comm );
+        double end_solidification_print_time = MPI_Wtime();
+        if ( mpi_rank_ == 0 )
+            std::cout << "Solidification data written in "
```

Comment:
This is useful, but this seems like a strange one-off output. Can we write solver/init time as well?

Reply:
I guess I didn't think of that since we already print solver time assuming the `total_monitor_steps` input isn't 0, and the init time has been insignificant compare to the run/write times. A timers input class like ExaCA has would be the long term solution here, but I didn't want to dive into that rabbit hole here given that I just needed the write times for the ExaCA chained vs coupled performance comparisons for the paper

Reply:
I totally forgot that was buried in the inputs file. In that case I would prefer we pick either `std::chrono` or `MPI_Wtime`, but I agree not diving too deep in this